src/test/java/org/alfresco/repo/search/impl/solr/SolrStoreMappingWrapperTest.java

```diff
@@ -134,36 +139,47 @@
     @Before
     public void init()
     {
+        doReturn("https").when(protocol).getScheme();
+
         doReturn("common").when(hostConfigurationCommon).getHost();
         doReturn(999).when(hostConfigurationCommon).getPort();
-        
+        doReturn(protocol).when(hostConfigurationCommon).getProtocol();
+
         doReturn("host").when(hostConfiguration1).getHost();
         doReturn(999).when(hostConfiguration1).getPort();

```

Comment:
Is it deliberate that you're omitting the protocol for hostConfiguration1?  Perhaps you could add a comment about this?